fili-core/src/main/java/com/yahoo/bard/webservice/async/workflows/DefaultAsynchronousWorkflowsBuilder.java

```diff
@@ -113,12 +113,14 @@ public AsynchronousWorkflows buildAsynchronousWorkflows(
         // JobRow has been stored because otherwise we have a race condition where the version of the JobRow with
         // a success/failure status could be written to the store first, and then immediately overwritten by the
         // original JobRow, leaving the JobRow in a state of "pending" for its lifetime.
-        Observable<JobRow> jobRowUpdatedNotification = buildUpdateRowChain(
+        ConnectableObservable<JobRow> jobRowUpdatedNotification = buildUpdateRowChain(
                 jobRowStoredNotification,
                 preResponseStoredNotification,
                 errorHandlingPreResponseEmitter,
                 jobMetadata
-        );
+        )
+                .replay(1);
+        jobRowUpdatedNotification.connect();
```

Comment:
The other notifications are hot, and since this observable relies on hot observables, I felt it was safest to also make it hot.